src/seedu/addressbook/data/person/printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+interface Printable {
+	public static final String WORD = "abc";
+	public static String getPrintableString (String WORD){
```

Comment:
I think you have missed the point of the concept of an interface in java. Yes java 8 allows you to implement a method. But in this exercise you were suppose only to define a method in the interface and then do the implementation in the respective classes.